README.md

```diff
@@ -28,3 +28,56 @@ async def main():
 
 asyncio.run(main())
 ```
+
+## Development
+
+Example script for Linux and Python 3.9 how to run everything.
+
+```sh
+# Checkout the repository
+git clone https://github.com/CorantGmbH/aioairq
+cd aioairq
+
+# Create a virtual environment
+python3.9 -m venv .venv
+. .venv/bin/activate
+
+# Install dependencies
+pip install -e .
+
+# You can now create a script (see example above) and run it
+vim hello-air-q.py
+python hello-air-q.py
+
+# Optional: Testing
+
+# Install optional dependencies for testing
+#
+# You find those in the pyproject.toml.
+# Be sure to install those separately, otherwise you'll install
+# pytest 8.0.0 which pytest-asyncio does not like.
```

Comment:
I am not sure I can reproduce this problem:
```
$ pip install pytest pytest-asyncio
Collecting pytest
  Using cached pytest-8.0.0-py3-none-any.whl.metadata (7.8 kB)
Collecting pytest-asyncio
  Using cached pytest_asyncio-0.23.4-py3-none-any.whl.metadata (3.9 kB)
Collecting iniconfig (from pytest)
  Using cached iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
Collecting packaging (from pytest)
  Using cached packaging-23.2-py3-none-any.whl.metadata (3.2 kB)
Collecting pluggy<2.0,>=1.3.0 (from pytest)
  Using cached pluggy-1.4.0-py3-none-any.whl.metadata (4.3 kB)
Collecting pytest
  Using cached pytest-7.4.4-py3-none-any.whl.metadata (7.9 kB)
Using cached pytest_asyncio-0.23.4-py3-none-any.whl (17 kB)
Using cached pytest-7.4.4-py3-none-any.whl (325 kB)
Using cached pluggy-1.4.0-py3-none-any.whl (20 kB)
Using cached packaging-23.2-py3-none-any.whl (53 kB)
Installing collected packages: pluggy, packaging, iniconfig, pytest, pytest-asyncio
Successfully installed iniconfig-2.0.0 packaging-23.2 pluggy-1.4.0 pytest-7.4.4 pytest-asyncio-0.23.4
```
As I see it, pip attempts to install 8.0.0 at first, but then resolves the conflict following the requirements of `pytest-asyncio`. How did you get into a conflict?

Reply:
No idea, I just noticed when installing both together `pip` is too stupid to resolve it.
I get an error it's the wrong version, but it does not correct it:
```sh
$ pip install pytest pytest-asyncio
Collecting pytest
  Using cached pytest-8.0.0-py3-none-any.whl (334 kB)
Collecting pytest-asyncio
  Using cached pytest_asyncio-0.23.4-py3-none-any.whl (17 kB)
Collecting packaging
  Using cached packaging-23.2-py3-none-any.whl (53 kB)
Collecting tomli>=1.0.0; python_version < "3.11"
  Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
Collecting pluggy<2.0,>=1.3.0
  Using cached pluggy-1.4.0-py3-none-any.whl (20 kB)
Collecting exceptiongroup>=1.0.0rc8; python_version < "3.11"
  Using cached exceptiongroup-1.2.0-py3-none-any.whl (16 kB)
Collecting iniconfig
  Using cached iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
ERROR: pytest-asyncio 0.23.4 has requirement pytest<8,>=7.0.0, but you'll have pytest 8.0.0 which is incompatible.
Installing collected packages: packaging, tomli, pluggy, exceptiongroup, iniconfig, pytest, pytest-asyncio
Successfully installed exceptiongroup-1.2.0 iniconfig-2.0.0 packaging-23.2 pluggy-1.4.0 pytest-8.0.0 pytest-asyncio-0.23.4 tomli-2.0.1
```
ending up with 8.0.0, which does not work.

However, when doing it one by one, `pip` again downloades 8.0.0, but then downgrades to the required 7.x.

```sh
$ pip install pytest
Collecting pytest
  Using cached pytest-8.0.0-py3-none-any.whl (334 kB)
Collecting tomli>=1.0.0; python_version < "3.11"
  Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
Collecting packaging
  Using cached packaging-23.2-py3-none-any.whl (53 kB)
Collecting pluggy<2.0,>=1.3.0
  Using cached pluggy-1.4.0-py3-none-any.whl (20 kB)
Collecting iniconfig
  Using cached iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
Collecting exceptiongroup>=1.0.0rc8; python_version < "3.11"
  Using cached exceptiongroup-1.2.0-py3-none-any.whl (16 kB)
Installing collected packages: tomli, packaging, pluggy, iniconfig, exceptiongroup, pytest
Successfully installed exceptiongroup-1.2.0 iniconfig-2.0.0 packaging-23.2 pluggy-1.4.0 pytest-8.0.0 tomli-2.0.1

$ pip install pytest-asyncio
Collecting pytest-asyncio
  Using cached pytest_asyncio-0.23.4-py3-none-any.whl (17 kB)
Collecting pytest<8,>=7.0.0
  Using cached pytest-7.4.4-py3-none-any.whl (325 kB)
Requirement already satisfied: tomli>=1.0.0; python_version < "3.11" in ./.venv/lib/python3.9/site-packages (from pytest<8,>=7.0.0->pytest-asyncio) (2.0.1)
Requirement already satisfied: pluggy<2.0,>=0.12 in ./.venv/lib/python3.9/site-packages (from pytest<8,>=7.0.0->pytest-asyncio) (1.4.0)
Requirement already satisfied: iniconfig in ./.venv/lib/python3.9/site-packages (from pytest<8,>=7.0.0->pytest-asyncio) (2.0.0)
Requirement already satisfied: exceptiongroup>=1.0.0rc8; python_version < "3.11" in ./.venv/lib/python3.9/site-packages (from pytest<8,>=7.0.0->pytest-asyncio) (1.2.0)
Requirement already satisfied: packaging in ./.venv/lib/python3.9/site-packages (from pytest<8,>=7.0.0->pytest-asyncio) (23.2)
Installing collected packages: pytest, pytest-asyncio
  Attempting uninstall: pytest
    Found existing installation: pytest 8.0.0
    Uninstalling pytest-8.0.0:
      Successfully uninstalled pytest-8.0.0
Successfully installed pytest-7.4.4 pytest-asyncio-0.23.4
```
ending up with the 7.4.4

---

I rather have an additional info to help somebody, than having them guess how to solve :wink: